docs/content/Contribution guidelines/operations/versioned-things.md

```diff
@@ -0,0 +1,142 @@
+- `v2alpha1`, for the Kubernetes API of KubeStellar releases 0.8 and later.
+
+## Syncer container image
+
+The syncer is KubeStellar's agent in WECs, and is based on a
+Kubernetes Pod that uses a container image at
+`quay.io/kubestellar/syncer`. This container image is built from
+contents of [the KubeStellar GitHub
+repo](https://github.com/kubestellar/kubestellar) focused by
+[ko](https://github.com/ko-build/ko) on the syncer binary. This image
+is normally built using the `make` target named
+"build-kubestellar-syncer-image. The syncer image is referenced in the
+output generated by the `kubectl kubestellar syncer-gen` command,
+which is invoked by `kubectl kubestellar prep-for-syncer`, which is
+invoked by `kubectl kubestellar prep-for-cluster`. The `kubectl
+kubestellar prep-for-syncer` script holds the default value for the
+tag to use in the reference to the syncer image.
+
+Following are some recent tags and what they correspond to.
+
+- `v0.9.0`, for the git commit with that tag (i.e., that release).
+
+- `v0.8.0`, for the git commit with that tag (i.e., that release).
+
+- `pr-1103` = `git-ae64d5722-clean`, incorporating [PR
+  1105](https://github.com/kubestellar/kubestellar/pull/1103), the
+  latest development in the release-0.7 line.
+
+## Core container image
+
+The core container runs the kcp server and the KubeStellar core
+controllers and initializes the kcp workspaces. It is built from some
+contents of [the KubeStellar GitHub
+repo](https://github.com/kubestellar/kubestellar), specifically
+excluding some that are only need by users outside this
+container. This image is normally built using the `make` target
+"kubestellar-image". This image is referenced from the core Helm
+chart, which can be used directly from the Helm repo and also is used
+by the `kubectl kubestellar deploy` command (which has a fixed image
+tag in its source, which is edited when it is time to advance that
+tag).
+
+Following are some recent tags and what they correspond to.
+
+- `release-0.9`, for the release 0.9 line of development; currently
+  this points to the same image as `v0.9.0`
+  but may move in the future.
+
+- `v0.9.0` = `v0.9.0-alpha.1.git1f0e2cc-clean`, for the v0.9.0 release.
+
+## KubeStellar core Helm chart
+
+This Helm chart describes a deployment of the KubeStellar core
+components (kcp server, KubeStellar controllers, initialization) as
+workload for a Kubernetes cluster. A Helm chart has both a version
+number for the chart itself and a version number for the "application"
+it installs. The `kubectl kubestellar deploy` command uses the
+adjacent copy of this Helm chart, and users can directly reference a
+copy in the Helm repo.
+
+Following are the versions of some recent editions of the chart.
+
+| Chart Version | App Version | What it is |
+| ------------- | ----------- | ---------- |
+|       1       |     v0.9.0  | KubeStellar release-0.9 branch |
+|     0.4.0     |     v0.4.0  | KubeStellar release-0.7 branch |
```

Comment:
the helm chart at helm.kubestellar.io does not serve 0.4.0.  we can remove this reference.  Also, the current version of the chart is '2'.  '1' no longer exists as it was incorrectly built.

Reply:
Updated. We have some confusion (detailed in Slack).

Reply:
_after_ instantiating the Helm chart from the kubestellar/helm repo, what does `helm list` say about the version of the chart that is running in the cluster?

Reply:
You did not ask it to look in the right namespace.